comixed-frontend/src/assets/i18n/en/app.json

```diff
@@ -90,6 +90,8 @@
       "no-description": "No description available...",
       "delete-comic-title": "Delete This Comic?",
       "delete-comic-question": "Are you sure? This will not delete the file, but only remove it from the library.",
+      "confirm-logout-title": "Logout",
```

Comment:
I'm not sure we would want to put the logout confirmation message into the library i18n namespace. Can we instead move this to the "global" namespace under a new collection called "confirmations"? Then set the two parts as "header" and "message" entries in there, which is in line with how other confirmations are translated:

```
  "global": {
    "errors": {
      "form-field-required": "{name} is required."
    },
    "tooltip": {
      "unprocessed-comic": "This comic is scheduled for processing.",
      "deleted-comic": "This comic is marked for deletion."
    },
   "confirmation": {
        "logout": {
            "header": "Logout",
            "message": "Are you sure you want to log out?"
        }
    }
  },
```

Reply:
Modification done